web/src/app/Login/shared/login.serviceApp.ts

```diff
@@ -4,13 +4,12 @@ import { Observable } from 'rxjs/Observable';
 import 'rxjs/Rx';
 import { HttpClientModule, HttpClient } from '@angular/common/http';
 import { LoginService, RoleService } from '../../webapi/services';
-
-
+import { UserService } from '../../webapi/services';
 
 @Injectable()
 export class LoginServiceApp {
 
-    constructor(private http: HttpClient, private apiLoginService: LoginService, private apiRoleService: RoleService) { }
+    constructor(private http: HttpClient, private apiLoginService: LoginService, private apiForgotpasswordService: UserService, private apiRoleService: RoleService) { }
```

Comment:
remove them private apiForgotpasswordService: UserService, private apiRoleService: RoleService
or mover to corresponding serices